submit-web/src/components/Submission/InternalDocuments/index.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Link as MuiLink,
+  TableCell,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import { BCDesignTokens } from "epic.theme";
+import { StaffSubmission, SUBMISSION_STATUS } from "@/models/Submission";
+import { Unless } from "react-if";
+import {
+  SubmissionItemTableCell,
+  PackageTableRow,
+} from "../SubmissionItemTableRow";
+import DocumentRow from "../DocumentRow";
+
+export default function InternalDocuments({
+  submissions,
+}: {
+  submissions: Array<StaffSubmission>;
+}) {
+  const actionLabel = "Add Documents";
+  const internalDocuments = submissions.flatMap((submission) =>
```

Comment:
the internal documents aren't submissions. They should be taken from the SubmissionItem